DuckDuckGo/FeedbackPickerViewController.swift

```diff
@@ -115,9 +115,9 @@ class FeedbackPickerViewController: UITableViewController {
         }
         
         let theme = ThemeManager.shared.currentTheme
-        cell.contentView.backgroundColor = theme.tableCellBackgroundColor
```

Comment:
When using 'selectedBackgroundView', in order to make sure cell's highlighted state work properly `contentView` should be transparent and `backgroundColor` should be used instead. Otherwise deselect animation has glitches.